config/formats.js

```diff
@@ -562,6 +562,36 @@ let Formats = [
 		ruleset: ['Pokemon', 'Standard NEXT', 'Team Preview'],
 		banlist: ['Uber'],
 	},
+	{
+		name: '[Gen 7] Shitmons',
+		threads: [
+			`&bullet; <a href="http://spo.ink/shitmons">Shitmons</a>`,
+		],
+
+		gameType: 'doubles',
```

Comment:
you need `mod: 'gen7',` above this line and `maxForcedLevel: 666,` below this line